src/Domain/User/ValueObject/Auth/HashedPassword.php

```diff
@@ -45,6 +49,10 @@ private static function hash(string $plainPassword): string
     {
         Assertion::minLength($plainPassword, 6, 'Min 6 characters password');
 
+        if (self::isTestSuite()) {
+            return $plainPassword;
+        }
+
```

Comment:
Why is this needed?

Reply:
https://blackfire.io/profiles/40247026-d471-4234-902c-e56990cea0f9/graph?settings%5Bdimension%5D=wt&settings%5Bdisplay%5D=landscape&settings%5BtabPane%5D=nodes&selected=&callname=main()

bcrypt cost in test suite is 1, too high 2, unnecessary

Reply:
Ok 👍🏼 . ( Link doesn't work 😞 )

Reply:
![Screenshot 2020-09-21 at 13 19 43](https://user-images.githubusercontent.com/4022187/93760915-2e502480-fc0d-11ea-9636-b9e3d4b42325.png)

Reply:
10+5s just encrypting and decrypting